src/docs/product/cli/releases.mdx

```diff
@@ -15,8 +15,7 @@ Because releases work on projects you will need to specify the organization and
 
 Releases are created with the `sentry-cli releases new` command. It takes at the very least a version identifier that uniquely identifies the releases. There are a few restrictions -- the release name cannot:
 
-- contain newlines or spaces
-- use a forward slash ("/"), back slash ("\"), period ("."), or double period ("..")
+- use a forward slash ("/")
```

Comment:
```suggestion
- be (".") or ("..") or the word ("latest")
- contain a forward slash ("/")
```

Reply:
Please also update the common file for Releases common/configuration/releases.mdx with this clarification.